app/domain/output_document.rb

```diff
@@ -3,10 +3,84 @@ class OutputDocument
 
   attr_accessor :appointment_summary
 
+  delegate :id, :guider_name, :income_in_retirement, :continue_working, :unsure,
+           :leave_inheritance, :wants_flexibility, :wants_security,
+           :wants_lump_sum, :poor_health,
+           to: :appointment_summary
+
+  delegate :address_line_1, :address_line_2, :address_line_3, :town, :county,
+           :postcode,
+           to: :appointment_summary, prefix: :attendee
+
   def initialize(appointment_summary)
     @appointment_summary = appointment_summary
   end
 
+  def attendee_name
+    "#{appointment_summary.title} #{appointment_summary.first_name} #{appointment_summary.last_name}"
```

Comment:
First name is optional, so would end up with double space like `Mr  Garner`. There is also a chance the first name and last name attributes could have whitespace as I don't believe they are sanitised before write. However, Rails has your back!

```rb
"#{appointment_summary.title} #{appointment_summary.first_name} #{appointment_summary.last_name}".squish
```